kazoo/retry.py

```diff
@@ -133,10 +133,10 @@ def __call__(self, func, *args, **kwargs):
             except ConnectionClosedError:
                 raise
             except self.retry_exceptions:
+                self._attempts += 1
                 # Note: max_tries == -1 means infinite tries.
```

Comment:
Do you see a way to test this case?

Reply:
I am not sure what you mean: what 'case'?
Do you mean that the exception is in the `self.exceptions` set?

Reply:
I was thinking of the not `Note: max_tries == -1 means infinite tries.`